### DISCUSSION TITLE
LTS 4 Files no longer available?

### DISCUSSION THREAD

**Discussion by drickles** [MustardChef/WSABuilds#444] [General]
For some reason non of the current links have the LTS4 files in it. has that project been scrapped or am i not looking for it right?

**Comment by MustardChef** [comment]
Just checked all links of the LTS4 releases in readme and they are all working and active. This means that there is an issue on your end. Try using a VPN.

**Reply by drickles** [reply]
In the link do download LTS4 only LTS3 files (v2311.40000.5.0) files can be found. LTS4 (v2407.40000.0.0) files cant be found.
